rasa/nlu/utils/__init__.py

```diff
@@ -56,13 +64,42 @@ def write_to_file(filename: Text, text: Any) -> None:
 
 
 def build_entity(
-    start: int, end: int, value: Text, entity_type: Text, **kwargs: Dict[Text, Any]
+    start: int,
+    end: int,
+    value: Text,
+    entity_type: Text,
+    role: Optional[Text] = None,
+    group: Optional[Text] = None,
+    **kwargs: Dict[Text, Any],
```

Comment:
The type is wrong. Although kwargs are a dictionary, you give it the type of the arguments. Should we get rid of this `kwargs` argument? It's not used anywhere
```suggestion
    **kwargs: Any,
```

Reply:
it is used to update the entity dict. Theoretically you can add any other key value pairs to an entity.